R/util.R

```diff
@@ -610,3 +610,46 @@ abbreviate <- function(x, len = round(getOption("width", 80) * 0.8)) {
   }
   x
 }
+
+handle_missing_packages <- function(missing_packages) {
+  ## collapse vector to packages to string "c('pckg_1','pckg_2')"
+  vector_packages <- sprintf("install.packages(c(%s))",
```

Comment:
we have an `squote` function for this job already - please use:

```
sprintf("install.packages(c(%s))", paste(squote(missing_packages), collapse = ", "))
``` 

(note space after comma in the `collapse`)